tests/python/mock_jira.py

```diff
@@ -730,11 +1632,15 @@ def search_assignable_users_for_issues(self, name, startAt=0, maxResults=20, *ar
             # Mock Jira REST api bug
             return []
 
+        options = {"deployment_type": "Cloud" if self.is_jira_cloud else "Server"}
+
         # Create a list of users.
-        users = [User(None, None, JIRA_USER_2)] * maxResults + [User(None, None, JIRA_USER)]
+        users = [User(options, None, JIRA_USER_2)] * maxResults + [
+            User(options, None, JIRA_USER)
+        ]
```

Comment:
This fixes an issue with the tests. Since we modified the Jira API to check whether the service was on "Cloud" or "Server", we now rely on the options being passed through to the User instantiation.